libs/feature/search/src/lib/favorites/favorite-star/favorite-star.component.ts

```diff
@@ -33,17 +47,55 @@ export class FavoriteStarComponent {
   record_: MetadataRecord
   favoriteCount: number | null
   loading = false
+  DEFAULT_GN4_LOGIN_URL = `/geonetwork/srv/${
+    LANG_2_TO_3_MAPPER[this.translateService.currentLang]
+  }/catalog.signin?redirect=`
+  loginUrl = `${
+    this.optionalLoginUrl ? this.optionalLoginUrl : this.DEFAULT_GN4_LOGIN_URL
+  }${window.location}`
+  loginMessage = this.translateService.instant(
+    'favorite.not.authenticated.tooltip',
+    {
+      link: this.loginUrl,
+    }
+  )
+  @ViewChild(StarToggleComponent, { read: ElementRef })
+  starToggleRef: ElementRef
+  subscription: Subscription
 
   get hasFavoriteCount() {
     return this.favoriteCount !== null
   }
 
   constructor(
+    @Optional()
+    @Inject(LOGIN_URL)
+    private optionalLoginUrl: string,
     private favoritesService: FavoritesService,
     private authService: AuthService,
-    private changeDetector: ChangeDetectorRef
+    private changeDetector: ChangeDetectorRef,
+    private translateService: TranslateService
   ) {}
 
+  ngAfterViewInit(): void {
+    this.subscription = this.isAnonymous$.subscribe((anonymous) => {
+      if (anonymous) {
+        tippy(this.starToggleRef.nativeElement, {
```

Comment:
I think next time we need a tooltip we should be careful to factorize this into a separate directive/component, to keep in mind for later :)